exercises/nucleotide-count/description.md

```diff
@@ -1,25 +1,13 @@
-Given a DNA string, compute how many times each nucleotide occurs in the string.
+Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
 
-DNA is represented by an alphabet of the following symbols: 'A', 'C',
-'G', and 'T'.
-
-Each symbol represents a nucleotide, which is a fancy name for the
-particular molecules that happen to make up a large part of DNA.
-
-Shortest intro to biochemistry EVAR:
+The genetic language of every living thing on the planet is DNA. 
+DNA is a large molecule that is built from an extremely long sequence of individual elements called nucleotides. 
+4 types exist in DNA and these differ only slightly and can be represented as the following symbols: 'A' for adenine, 'C' for cytosine, 'G' for guanine, and 'T' thymine.
 
+Here is an analogy:
 - twigs are to birds nests as
-- nucleotides are to DNA and RNA as
-- amino acids are to proteins as
-- sugar is to starch as
-- oh crap lipids
-
-I'm not going to talk about lipids because they're crazy complex.
-
-So back to nucleotides.
-
-DNA contains four types of them: adenine (`A`), cytosine (`C`), guanine
-(`G`), and thymine (`T`).
+- nucleotides are to DNA as
+- legos are to lego houses as
+- words are to sentences as...
 
-RNA contains a slightly different set of nucleotides, but we don't care
-about that for now.
+In RNA the thymine is substituted with 'U' for uracil, while the remainder stay the same; A, U, C, G.
```

Comment:
I don't think this information is necessary for this exercise, and including it will cause confusion. A student would naturally ask "Why did you tell me this? Why do I need to know this?"

Reply:
Dealt with by #918